gapic/batch/staging.yaml

```diff
@@ -0,0 +1,6 @@
+common:
+  batch_apis: '*'
+  batch_languages: '*'
+  api_config_pattern: ${THISDIR}/../api/artman_*.yaml
+  artman_language_yaml: ${THISDIR}/../lang/common.yaml
+  batch_output_dir: ${REPOROOT}/client-library-staging/generated
```

Comment:
How about 'staging_output_dir' so that it can also be used when generating a single language x api? That would also require an artman option to make it put output into staging_output_dir instead of final_repo_dir. 

Sidenote, I think final_repo_dir is not terribly usable as it is set up now, and is inconsistent all over the place. So actually, I kind of like the idea of redirecting *everything* to staging_repo_dir, and leaving final_repo_dir dangling until we get it sorted out. At the same time, we could nuke the output_dir config option, and just fix it to one thing.